Source/CesiumRuntime/Private/CesiumFeatureIdAttribute.cpp

```diff
@@ -35,63 +40,79 @@ struct VertexCountFromAccessor {
 } // namespace
 
 FCesiumFeatureIdAttribute::FCesiumFeatureIdAttribute(
-    const Model& model,
-    const Accessor& featureIDAccessor,
-    int32 attributeIndex,
-    const FString& featureTableName)
-    : _featureTableName(featureTableName), _attributeIndex(attributeIndex) {
-  switch (featureIDAccessor.componentType) {
-  case CesiumGltf::Accessor::ComponentType::BYTE:
+    const Model& InModel,
```

Comment:
Out of curiosity, why `InModel` rather than just `Model`? I think it should be ok in most cases to have both a type and parameter named `Model`.

Reply:
I thought Visual Studio complained about that before, but it may have been Intellisense being wonky. I'll rename it to `Model` for consistency with the other variables.